src/vim.h

```diff
@@ -1576,10 +1576,10 @@ typedef UINT32_TYPEDEF UINT32_T;
 #endif
 
 /*
- * EMX doesn't have a global way of making open() use binary I/O.
+ * Cygwin doesn't have a global way of making open() use binary I/O.
```

Comment:
> Cygwin doesn't have a global way of making open() use binary I/O.

Actually, Cygwin can use binary I/O by linking `-lbinmode`:
https://cygwin.com/cygwin-ug-net/using-textbinary.html

If we use this, we can stop using `O_EXTRA`.